sections/products.liquid

```diff
@@ -25,19 +25,6 @@
 {%- if size > 0 -%}
   {%- assign show_carousel = section.settings.show_carousel -%}
 
-  {%- capture css_urls -%}
-    [
-      "{{ 'products.css' | asset_url }}",
-      "{{ 'product-card.css' | asset_url }}",
-      "{{ 'focal-image.css' | asset_url }}"
-      {%- if size > 1 and show_carousel -%}
-        , "{{ "carousel.css" | asset_url }}"
-      {%- endif -%}
-    ]
-  {%- endcapture -%}
-
-  {%- render 'section-styles-loading', css_urls: css_urls -%}
```

Comment:
@dabrorius the logic I was trying to move into the backend leveraging the schema properties

Reply:
To me building this kind of array using commas (e.g. `, "{{ "carousel.css" | asset_url }}"`) looks complicated and difficult to maintain in the long run. Looks like Liquid code should be simpler and not built for having this kind of logic?